metadata-builders/src/main/java/com/linkedin/metadata/builders/search/DataProcessIndexBuilder.java

```diff
@@ -1,8 +1,15 @@
 package com.linkedin.metadata.builders.search;
```

Comment:
could you also add some tests for this index builder? similar to `CorpGroupIndexBuilderTest.java`